### PR TITLE
Milestone 5

### PR DESCRIPTION
**Pull request by Oxford-G**
The final milestone of the calculator project. Added interaction to the buttons.

### CODE REVIEW

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 
@Oxford-G  Great job done on this milestone, almost all the project requirement is met,a commendable effort indeed, however, few changes are required:

### Highlights

- it is converted to a stateful component using class 👍🏻 
-  it has state initialized in constructor (screenshot) 👍🏻 
-  it has handleClick implemented in constructor 👍🏻 
- it passes `handleClick` as a `clickHandler` property to `ButtonPanel 👍🏻 
- it implements `handleClick` function 👍🏻 
-  it passes `handleClick` as `clickHandler` props to each `Button` 👍🏻 
-  it attaches `handleClick` to `onClick` event on Button 👍🏻 

### To Improve

- [ ] Your PR is not descriptive kindly add more content and for your subsequent projects always capitalized your commits
- [ ] Kindly handle to display `Division by zero` when number  `÷`  by  `0`.

### once specific changes are made kindly request for review

File: src/logic/operate.js [hunk 1]
```diff
@@ -12,7 +16,7 @@ function operate(numberOne, numberTwo, operation) {
     case '-':
       result = num1.minus(num2);
       break;
-    case 'X':
+    case 'x':
       result = num1.times(num2);
       break;
     case '÷':
```
Comment (bellom) [inline src/logic/operate.js]: As mentioned by the previous reviewer;

Division by zero is not handled properly yet. Let's fix this.
The reason is that when a division by zero happens, we should return an error message like `division error`

Divide a number by zero (e.g. 4 / 0)

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Oxford-G   :clap: 

# Status: Changes Required! ♻️ 

**Kindly attend to the issue mentioned by the previous reviewer.**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @Oxford-G, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] No change has been made since the last review. Kindly implement the changes suggested in the previous review. 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Slack: Oloo Moses
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by oloomoses — APPROVED** [review]
Hi @Oxford-G, 

Your milestone is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**